src/game_frontend/src/components/ui/Navbar/index.css

```diff
@@ -1,18 +1,19 @@
-/* Main navbar container */
+/* Main navbar container chNG */
+
 .top-nav-bar {
-    @apply absolute top-[72px] left-[64px] right-[64px] w-[1400px] h-[72px] flex items-center justify-between bg-[#D9CCC1] opacity-60 p-2;
+    @apply  top-[72px] left-[64px] right-[64px] max-w-[900px] w-auto mx-auto h-[72px] flex items-center justify-between bg-[#D9CCC1] opacity-60 p-2;
```

Comment:
Remove the whole top, left, right styles cause they are not applicable for containers with `position : relative`